diff -up qttools-opensource-src-5.6.2/src/qdoc/qdocindexfiles.cpp.QTBUG-43057 qttools-opensource-src-5.6.2/src/qdoc/qdocindexfiles.cpp
--- qttools-opensource-src-5.6.2/src/qdoc/qdocindexfiles.cpp.QTBUG-43057	2016-11-03 09:19:59.257547541 -0500
+++ qttools-opensource-src-5.6.2/src/qdoc/qdocindexfiles.cpp	2016-11-03 09:21:39.574157002 -0500
@@ -1412,7 +1412,7 @@ void QDocIndexFiles::generateIndexSectio
             const Aggregate* inner = static_cast<const Aggregate*>(node);
 
             NodeList cnodes = inner->childNodes();
-            std::sort(cnodes.begin(), cnodes.end(), Node::nodeNameLessThan);
+            qSort(cnodes.begin(), cnodes.end(), Node::nodeNameLessThan);
 
             foreach (Node* child, cnodes) {
                 generateIndexSections(writer, child, generateInternalNodes);
