diff -up qttools-opensource-src-5.1.1/src/linguist/lrelease/main.cpp.qmake-qt5 qttools-opensource-src-5.1.1/src/linguist/lrelease/main.cpp
--- qttools-opensource-src-5.1.1/src/linguist/lrelease/main.cpp.qmake-qt5	2013-08-25 13:04:37.000000000 -0500
+++ qttools-opensource-src-5.1.1/src/linguist/lrelease/main.cpp	2013-09-21 01:01:44.970539849 -0500
@@ -311,9 +311,9 @@ int main(int argc, char **argv)
             option.qmake_abslocation = QString::fromLocal8Bit(qgetenv("QMAKE"));
             if (option.qmake_abslocation.isEmpty())
 #ifdef QT_BOOTSTRAPPED
-                option.qmake_abslocation = binDir + QLatin1String("/qmake");
+                option.qmake_abslocation = binDir + QLatin1String("/qmake-qt5");
 #else
-                option.qmake_abslocation = app.applicationDirPath() + QLatin1String("/qmake");
+                option.qmake_abslocation = app.applicationDirPath() + QLatin1String("/qmake-qt5");
 #endif
             option.initProperties();
             QMakeParser parser(0, &evalHandler);
diff -up qttools-opensource-src-5.1.1/src/linguist/lupdate/main.cpp.qmake-qt5 qttools-opensource-src-5.1.1/src/linguist/lupdate/main.cpp
--- qttools-opensource-src-5.1.1/src/linguist/lupdate/main.cpp.qmake-qt5	2013-08-25 13:04:37.000000000 -0500
+++ qttools-opensource-src-5.1.1/src/linguist/lupdate/main.cpp	2013-09-21 01:00:41.474204348 -0500
@@ -824,7 +824,7 @@ int main(int argc, char **argv)
         ProFileGlobals option;
         option.qmake_abslocation = QString::fromLocal8Bit(qgetenv("QMAKE"));
         if (option.qmake_abslocation.isEmpty())
-            option.qmake_abslocation = app.applicationDirPath() + QLatin1String("/qmake");
+            option.qmake_abslocation = app.applicationDirPath() + QLatin1String("/qmake-qt5");
         option.initProperties();
         option.setCommandLineArguments(QDir::currentPath(),
                                        QStringList() << QLatin1String("CONFIG+=lupdate_run"));
